src/hook.rs

```diff
@@ -79,7 +81,7 @@ pub fn load_env(
         (None, None) => {
             return Ok(None);
         }
-        (Some(a), Some(t)) if a.hash == t.hash()? => {
+        (Some(a), Some(t)) if a.hash == t.hash()? && !force => {
             return Ok(None);
         }
         (_, _) => (),
```

Comment:
It looks to me like when `__shadowenv_data`, which contains the `active`, was unexported, the new shell scenario would have led to `(None, Some(t))`, caught in this default case.  This PR returns us to that case.  I do wonder though, if it also reintroduces what exporting `__shadowenv_data` in #67 fixed. @burke can you comment on the nature of the tmux problems so that @shanesmith can check whether they come back with this change?

Reply:
Yeah, the basic issue was:

1. User starts a shell, `dev cd shopify`, activates core env
2. User starts tmux. Tmux inherits core env, but not undo data
3. User moves to other project. Because the core env is applied, but the undo data is gone, shadowenv just merges this project's env into core's env.